docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contributing to Phonebook
+
+## What should I know before I get started?
```

Comment:
This section contains only developing instructions. I would move it to the developing part.
Or just swap the "What should I know before I get started?" part with "How can I contribute?"